### DISCUSSION TITLE
Option to install Chromium before exec

### DISCUSSION THREAD

**Discussion by colemanja91** [percy/cli#909] [Support] [answered]
Hi there! 
We recently ran into an issue in our usage of the Percy CLI. Due to some intermittent network issues, downloading Chromium was taking an extremely long time (in some of our builds, it was > 40 minutes). On top of that, we use parallel builds, so Chromium was being downloaded multiple times per workflow. 

We resolved this on our side by adding the following to our CI config in a bootstrap step which generates our yarn cache:

```
npx percy exec --parallel -- echo "Ensure Chromium installation"
```

It might be nice to help prevent others from getting caught in the same issue. Some possible solutions:
* Add the above CI command to docs (maybe a "CI Optimizations" section)
* Add a Chromium executable path option, i.e. `percy exec --parallel --chromium-exec /path/to/chromium -- ...`
* Add a setup step which downloads any dependencies and can be run independent, i.e. `percy setup`

If any of these sound viable, I'd be happy to help out and open a PR.

**Comment by Robdel12** [comment, marked as answer]
Hey @colemanja91! You could set the path to your own chromium if you'd like: https://docs.percy.io/docs/skipping-asset-discovery-browser-download or you could set an env var to have the chromium browser downloaded as apart of your dependencies: https://docs.percy.io/docs/caching-asset-discovery-browser-in-ci
